website/src/components/Messages/AdminMessageTable.tsx

```diff
@@ -0,0 +1,110 @@
+import { Avatar } from "@chakra-ui/avatar";
+import { Badge, Flex } from "@chakra-ui/layout";
+import { Tooltip } from "@chakra-ui/react";
+import { createColumnHelper } from "@tanstack/table-core";
+import { formatDistanceToNow, formatISO9075 } from "date-fns";
```

Comment:
I wonder if this is something that `next-i18n` already provides.

Reply:
From my knowledge, the answer is no.  I'm pretty sure we'd have to use some new library to do this so date-fns seems good enough